packages/table/test/indexTests.ts

```diff
@@ -0,0 +1,108 @@
+/**
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ * Licensed under the BSD-3 License as modified (the “License”); you may obtain a copy
+ * of the license at https://github.com/palantir/blueprint/blob/master/LICENSE
+ * and https://github.com/palantir/blueprint/blob/master/PATENTS
+ */
+
+import { expect } from "chai";
+
+import * as tableExports from "../src";
+
+describe("@blueprintjs/table", () => {
+    interface IKnownKeysMap {
+        [key: string]: boolean;
+    }
+
+    const knownKeys: IKnownKeysMap = {};
+
+    describe("Exports all expected values", () => {
```

Comment:
not really a fan of these tests; they're tautological and changes to exports in index files should be caught in code review

Reply:
Yeah, if there were some automated way to guarantee that every type and value mentioned in the docs were exported, that'd be nice; but with this approach here, you necessarily have to remember to write a test in order to guarantee that the thing you care about is exported, at which point you'll already have remembered that you need to export the thing.